src/components/Navbar.vue

```diff
@@ -1,55 +1,92 @@
 <template>
   <header>
     <div id="btnNav">
-      <div id="rawBtn">
-        <section id="rawBtnNav">
-          <input
-            type="button"
-            value="Cadastro"
-            class="rawBtnNav disabled"
-            disabled
-          />
+      <div id="btn">
+        <section id="btnNavRegister">
+
+          <button @click="btnCadastro" class="button btnClickCadastro" v-if="cadastroClick">
+            <router-link to="/register" class="click clickBtn">Cadastro</router-link>
+          </button>
+
+          <button @click="rawBntCadastro" class="button" v-else>
+            <router-link to="/register" class="rawBtnNav click">Cadastro</router-link>
+          </button>
```

Comment:
Sempre quando for fazer um redirect, é ideal usar o name da rota e não o caminho dela

Reply:
Continuarias sendo utilizado o `to=""`?? Porque eu mudei pro nome da rota mas ele direcionou errado

Reply:
Como fez?